server/konnectors/aprr.js

```diff
@@ -0,0 +1,202 @@
+'use strict';
+
+const request = require('request').defaults({
+  jar: true,
+  rejectUnauthorized: false,
+  followAllRedirects: true   // allow redirections
+});
+const moment = require('moment');
+const cheerio = require('cheerio');
+const baseKonnector = require('../lib/base_konnector');
+
+const filterExisting = require('../lib/filter_existing');
+const localization = require('../lib/localization_manager');
+const saveDataAndFile = require('../lib/save_data_and_file');
+const linkBankOperation = require('../lib/link_bank_operation');
+
+const log = require('printit')({
+  prefix: 'APRR',
+  date: true,
+});
+
+const fileOptions = {
+  vendor: 'APRR',
+  dateFormat: 'YYYYMMDD',
+};
+
+const Bill = require('../models/bill');
+
+// Konnector
+const connector = module.exports = baseKonnector.createNew({
+  name: 'APRR',
+  fields: {
+    login: 'text',
+    password: 'password',
+    folderPath: 'folder',
+  },
+  models: [Bill],
+  fetchOperations: [
+    getHiddenInputs,
+    logIn,
+    fetchBillingInfo,
+    parsePage,
+    customFilterExisting,
+    customSaveDataAndFile,
+    linkBankOperation({
+      log,
+      model: Bill,
+      identifier: 'APRR',
+      minDateDelta: 4,
+      maxDateDelta: 20,
+      amountDelta: 0.1,
+    }),
+    buildNotifContent,
+  ],
+});
+
+// Procedure to get hidden inputs
+function getHiddenInputs(requiredFields, bills, data, next) {
+  const url = 'https://espaceclient.aprr.fr/aprr/Pages/connexion.aspx';
+  const options = {
+    url,
+    method: 'GET',
+  };
+
+  connector.logger.info('Getting the hidden inputs...');
```

Comment:
do you mean instead of connector.logger.info ? what is the best practice (I don't really need the log object) ?